lib/Marlin/Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -448,36 +567,28 @@ void GcodeSuite::G28_no_parser(bool always_home_all, bool O, float R, bool S, bo
    */
   #if ENABLED(DUAL_X_CARRIAGE)
 
-    if (dxc_is_duplicating()) {
+    if (idex_is_duplicating()) {
+
+      TERN_(IMPROVE_HOMING_RELIABILITY, saved_motion_state = begin_slow_homing());
 
       // Always home the 2nd (right) extruder first
       active_extruder = 1;
-      homeaxis(X_AXIS
-        #if ENABLED(PRECISE_HOMING)
-          , 0.0f, false, !parser.seen('D')
-        #endif
-      );
+      homeaxis(X_AXIS);
 
       // Remember this extruder's position for later tool change
-      inactive_extruder_x_pos = current_position.x;
+      inactive_extruder_x = current_position.x;
 
       // Home the 1st (left) extruder
       active_extruder = 0;
-      homeaxis(X_AXIS
-        #if ENABLED(PRECISE_HOMING)
-          , 0.0f, false, !parser.seen('D')
-        #endif
-      );
+      homeaxis(X_AXIS);
```

Comment:
Looks like PRECISE_HOMING 'D' - do not calibrate feature is dropped here.

Reply:
As above. I'm not against the change itself, but I don't like how we specialize each call. Fixing this now would be a mistake though, so I'm waiting until all of 2.1.x is merged before we do cosmetic fixes of this sort.